web/src/beta/features/Editor/useLayers.ts

```diff
@@ -0,0 +1,90 @@
+import { useCallback, useMemo, useState } from "react";
+
+import { useLayersFetcher } from "@reearth/services/api";
+import { NLSLayer } from "@reearth/services/api/layersApi/utils";
+import { useT } from "@reearth/services/i18n";
+
+type Props = {
+  sceneId: string;
+  nlsLayers: NLSLayer[];
+};
+
+export type LayerAddProps = {
+  config?: any;
+  index?: any;
+  layerType: string;
+  sceneId: string;
+  title: string;
+  visible?: boolean;
+};
+
+export type LayerUpdateProps = {
+  layerId: string;
+  name: string;
+  visible?: boolean;
+};
+
+export default function ({ nlsLayers }: Props) {
+  const t = useT();
+  const { useAddNLSLayerSimple, useRemoveNLSLayer, useUpdateNLSLayer } = useLayersFetcher();
+  const [selectedLayerId, setSelectedLayerId] = useState<string | undefined>(undefined);
+
+  const selectedLayer = useMemo(() => {
+    return nlsLayers.length ? nlsLayers[0] : undefined;
+  }, [nlsLayers]);
```

Comment:
Shouldn't selected layer be the layer corresponding with the selectedLayerId??

Reply:
Done.